CONTRIBUTING.md

```diff
@@ -2,34 +2,42 @@
 
 ## Contributor License Agreements
 
-We'd love to accept your sample apps and patches! Before we can take them, we
-have to jump a couple of legal hurdles.
+Before you're able to contribute to this repository, you need to sign a
+Contributor License Agreement (CLA). You can follow the links below to
+fill out the appropriate CLA (individual, or corporate):
 
-Please fill out either the individual or corporate Contributor License Agreement
-(CLA).
-
-* If you are an individual writing original source code and you're sure you own
-  the intellectual property, then you'll need to sign an [individual
-  CLA](https://developers.google.com/open-source/cla/individual).
-* If you work for a company that wants to allow you to contribute your work,
-  then you'll need to sign a [corporate
-  CLA](https://developers.google.com/open-source/cla/corporate).
+* **[Individual
+  CLA](https://developers.google.com/open-source/cla/individual):** You are an individual writing original source code and own the intellectual property.
+* **[Corporate
+  CLA](https://developers.google.com/open-source/cla/corporate):** You work for a company that allows you to contribute your work.
 
 Follow either of the two links above to access the appropriate CLA and
 instructions for how to sign and return it. Once we receive it, we'll be able to
 accept your pull requests.
 
-## Contributing A Patch
+## Contributing a patch
 
-1. Submit an issue describing your proposed change to the repo in question.
-1. The repo owner will respond to your issue promptly.
-1. If your proposed change is accepted, and you haven't already done so, sign a
-   Contributor License Agreement (see details above).
-1. Fork the desired repo, develop and test your code changes.
+1. Submit an issue describing your proposed changes.
+1. The repo owner will respond to your issue promptly. Once accepted:
+1. Sign a Contributor License Agreement (see details above) if you haven't done so.
+1. Fork the repo, develop and test your code changes.
 1. Ensure that your code adheres to the existing style in the sample to which
-   you are contributing. Refer to the
-   [Google Cloud Platform Samples Style Guide]
-   (https://github.com/GoogleCloudPlatform/Template/wiki/style.html) for the
-   recommended coding standards for this organization.
+   you are contributing.
 1. Ensure that your code has an appropriate set of unit tests which all pass.
 1. Submit a pull request.
+
+## Samples requirements
+
+All new code sample needs the following requirements:
+- A GitHub Action workflow that tests the sample code. At minimum, this should
+  dry-run any container image or Terraform configs and pass without any errors.
```

Comment:
**Nitpick (feel free to ignore):**
What exactly does "dry-run" mean here?
Consider rephrasing this to something more actionable such as:
```
At minimum, this should build any container images and [`terraform validate`](https://developer.hashicorp.com/terraform/cli/commands/validate) any Terraform, and pass without any errors.
```

Reply:
Added more details!

Reply:
This is perfect. And more organized. Thanks, Olivier.